src/twisted/words/protocols/irc.py

```diff
@@ -2651,8 +2654,17 @@ def dataReceived(self, data):
 
     def lineReceived(self, line):
         if bytes != str and isinstance(line, bytes):
```

Comment:
Could you delete always-true condition while you're here? This is a Python 2 &rarr; 3 porting artifact.